src/material-experimental/mdc-checkbox/checkbox.scss

```diff
@@ -0,0 +1,42 @@
+@import '@material/checkbox/functions';
+@import '@material/checkbox/mixins';
+@import '@material/form-field/mixins';
+@import '@material/ripple/variables';
+@import '../mdc-helpers/mdc-helpers';
+
+@include mdc-checkbox-without-ripple($query: $mat-base-styles-query);
+@include mdc-form-field-core-styles($query: $mat-base-styles-query);
+
+.mat-mdc-checkbox {
+  // stylelint-disable selector-class-pattern
+
+  .mdc-checkbox:hover
+      .mdc-checkbox__native-control:not([disabled]) ~ .mdc-checkbox__background::before {
+    opacity: map-get($mdc-ripple-dark-ink-opacities, hover);
+    transform: scale(2.75, 2.75);
+    transition: mdc-checkbox-transition-enter(opacity, 0, 80ms),
+                mdc-checkbox-transition-enter(transform, 0, 80ms);
+  }
+
+  .mdc-checkbox:hover .mdc-checkbox__native-control:focus ~ .mdc-checkbox__background::before {
+    opacity: map-get($mdc-ripple-dark-ink-opacities, hover) +
+                 map-get($mdc-ripple-dark-ink-opacities, focus);
+  }
+
+  .mat-ripple-element {
+    opacity: map-get($mdc-ripple-dark-ink-opacities, press);
+  }
+
+  &._mat-animation-noopable {
```

Comment:
Since we have control of the template, we could target only the particular elements, rather than everything.

Reply:
That's true, though I don't really know what we'd gain. It would be more likely to break if MDC adds animations, and it would result in larger CSS size.